staging/src/k8s.io/client-go/examples/third-party-resources/main.go

```diff
@@ -20,17 +20,26 @@ package main
 import (
 	"flag"
 	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/golang/glog"
```

Comment:
As feiskyer suggested, run `./hack/update-bazel.sh?`. It will help to update `staging/src/k8s.io/client-go/examples/third-party-resources/BUILD`.